src/entities/errors.ts

```diff
@@ -216,6 +218,7 @@ export enum BackendErrorCode {
   BackendInvalidOperationSession = 7877,
   BackendPurchaseCannotBeCompleted = 7878,
   BackendEmailIsRequired = 7879,
+  BackendSubscriberNotFound = 7259,
```

Comment:
NABD but should we move this a bit above so the codes are ordered?

Reply:
Done